modules/fts/pages/fts-accessing-managing-full-text-indexes.adoc

```diff
@@ -1,7 +1,10 @@
 = Accessing and Managing Full Text Indexes
 
 [abstract]
-Full Text Indexes are different from the _Global_ indexes that are accessed under the *Indexes* tab in the left-hand navigation panel of the Couchbase Web Console.
+Full Text indexes, which are available under the *Indexes* tab of the Couchbase Web Console, are another form of the index provided by the Search service.
```

Comment:
Full Text Indexes are NOT available under the "Indexes" tab - they can be viewed within the "Search" tab.

Reply:
fixed